.devcontainer/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/java/.devcontainer/base.Dockerfile
+
+# [Choice] Java version: 11, 15
+ARG VARIANT="15"
+FROM mcr.microsoft.com/vscode/devcontainers/java:0-${VARIANT}
+
+# [Option] Install zsh
+ARG INSTALL_ZSH="true"
```

Comment:
I know you override it but perhaps we should always set this to false by default in the Dockerfile too.  I think I need to make that change for my superset devcontainer as well.

Reply:
Sure, I don't mind. The reason is the same as in my previous comment. It's a carry-over and just a default. It only matters if it's not specified in the build args.